packages/flutter/lib/src/cupertino/refresh.dart

```diff
@@ -11,8 +11,8 @@ import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 
 import 'activity_indicator.dart';
-import 'colors.dart';
-import 'icons.dart';
+
+const double _activityIndicatorRadius = 14.0;
```

Comment:
ah, thanks. Have updated this constant, plus also the margin one you mentioned further down.

The default radius for the activity indicator has been set to 10 since the very initial commit (and it used as a default in the constructor, so I guess forms part of the public API)... any apps that just use the activity indicator (outside of the pull-to-refresh context) would expect for that to remain the same. 

In the pull-to-refresh context, though, the default radius is set to 14 (since the original commit).

Reply:
Cool! Thanks for checking!